en/08_Changelogs/5.2.0.md

```diff
@@ -215,6 +216,14 @@ SilverStripe\ErrorPage\ErrorPage:
     - 500
 ```
 
+### Versioned status badges for elemental {#elemental-badges}
+
+Elemental content blocks now feature versioned status badges which adds a textual indicator of whether the block is in draft and/or a modified state. This textual indicator is in addition to the dot icon indicator that was already there. This functionality is only enabled if the [`Versioned`](api:SilverStripe\Versioned\Versioned) extension is enabled which it is by default.
+
+![elemental-versioned-badge](../_images/changeslogs/5.2.0/elemental-badge.png)
```

Comment:
Path is wrong, doesn't render. I think this is all that's needed but double check before just copying this:
```suggestion
![elemental-versioned-badge](../_images/changelogs/5.2.0/elemental-badge.png)
```

Reply:
Updated

Reply:
Updated

Reply:
Updated